autopush/websocket.py

```diff
@@ -935,16 +971,37 @@ def finish_webpush_notifications(self, notifs):
 
         # Send out all the notifications
         now = int(time.time())
+        messages_sent = False
         for notif in notifs:
             # If the TTL is too old, don't deliver and fire a delete off
             if notif.expired(at_time=now):
-                self.force_retry(self.ps.message.delete_message, notif)
-                continue
+                if not notif.sortkey_timestamp:
+                    # Delete non-timestamped messages
+                    self.force_retry(self.ps.message.delete_message, notif)
+
+                # The line above runs, but coverage is confused on this
```

Comment:
The 'continue' statement gets flagged by coverage as not being covered. No idea why, it *has* to run logically given that the lines below are fine.